lib/types.ts

```
@@ -22,56 +22,58 @@ export type WorkflowSettings = {
+  /**
+   * Exposes access to the kinde environment variables
+   */
+  "kinde.env"?: {};
+  /**
+   * Exposes access to the kinde environment variables
+   */
+  "kinde.auth"?: {};
+  /**
+   * Exposes access to the kinde localization
+   */
+  "kinde.localization": {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)